tests/conftest.py

```diff
@@ -62,6 +63,7 @@ def verify_cleanup():
 def hass(loop):
     """Fixture to provide a test instance of HASS."""
     hass = loop.run_until_complete(async_test_home_assistant(loop))
+    loop._thread_ident = threading.get_ident()
```

Comment:
We are not in the thread that is the loop here. We are in a fixture which is executed synchronously. That's why the previous line actually calls `loop.run-until_complete`